articles/time_to_say_goodby.md

```diff
@@ -0,0 +1,55 @@
+---
+authorName: Sabrina Lange
+authorGithubUsername: sweinzie
+issue: 8
+title: Time To Say Goodby
+---
+# {{page.title}}
+
+Time to Say Goodbye - What Should be Considered for a Successful Project Completion? 
+
+Setting up a project in an efficient way is not an easy task: To share visions of the project or of the project manager, to ensure that all are aligned and have a common understanding of how the project will be managed. It is nowadays a good practice of most project managers to have a kick off workshops, not only 1h but one day with with teambuilding and vision sharpening, not only once but depending on the project duration in regular intervals. 
+
+But how does it look like at the end of the project? Is everyone aware of how long he will be involved in the project? What are the tasks when the project is coming to an end? When will be the last working day? What will happen with the project deliveries after project closing? Who is going to ensure operations starting when and how?
+
+This is what unfortunately I observe within the most project setups, also confirmed by my colleagues:
+
+Sometimes Ops and Support were involved too late or not in an efficient way due to time constrains or other reasons. Therefore, they are not enabled to take over tasks and responsibility.
+ 
+On the other hand, transfer to operation is not proceeding because the project team is still receiving all new requests and tries to handle these. 
+ 
+And as there might be a certain kind of uncertainty in terms of how things are going and how much longer we will be involved in the project, people are happy to be contacted and needed.
```

Comment:
In einer Bullet-Point Liste würde ich den Satz umkehren:
"People are happy to be contacted and needed, because there might be a certain kind of uncertainty in terms of how things are going and how much longer we will be involved in the project"

Mein erster Gedanke war: "Jede Veränderung bringt Unsicherheit." - das steht noch nirgends so explizit, allenfalls kannst du das reinnehmen?